src/plugins/discover/public/application/view_components/canvas/index.tsx

```diff
@@ -43,39 +41,6 @@ export default function DiscoverCanvas({ setHeaderActionMenu, history }: ViewPro
     bucketInterval: {},
   });
 
-  const [isCallOutVisible, setIsCallOutVisible] = useState(
-    localStorage.getItem(KEY_SHOW_NOTICE) !== 'false'
```

Comment:
it's probably unnecessary to clear out this key if it does exist correct?

Reply:
Yeah, thats overkill :)